courses/fundamentals_of_ada/065_type_derivation.rst

```diff
@@ -84,7 +84,13 @@ General Rule For a Primitive
         + As a parameter
         + As its return type (for :ada:`function`)
 
-   - The **freeze-point** is below `S` declaration
+   - In scope of `S` declaration
+   - `S` is above :dfn:`freeze-point`
+
+* Rule of thumb
+
+    - Declare **all** of the type's primitives right after itself
```

Comment:
"after itself" should be "after the type itself"

Reply:
Done